x-pack/plugins/alerting/server/routes/get_rule_execution_log.ts

```diff
@@ -72,6 +72,11 @@ export const getRuleExecutionLogRoute = (
       verifyAccessAndContext(licenseState, async function (context, req, res) {
         const rulesClient = (await context.alerting).getRulesClient();
         const { id } = req.params;
+        if (id === '*') {
```

Comment:
I see so we're using this `*` to denote all rules (I see later in the PR where we're passing in `*` to the event log components). I'm wondering if we should use something like an enum or another prop to indicate that we want all rules? I feel like the component consumer needs to know that the ruleId prop can also be a *, which doesn't seem to be an existing convention. 

curious to hear your thoughts :slightly_smiling_face:

Reply:
I think it would make sense to have a separate route for global logs.